src/merchantPayment/FeeContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import BigNumber from 'bignumber.js'
+import React, { useEffect } from 'react'
+import { StyleSheet, View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+import FeeDrawer from 'src/components/FeeDrawer'
+import TokenTotalLineItem from 'src/components/TokenTotalLineItem'
+import { estimateFee, FeeType } from 'src/fees/reducer'
+import { feeEstimatesSelector } from 'src/fees/selectors'
+import { useCurrencyToLocalAmount } from 'src/localCurrency/hooks'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { BASE_TAG } from 'src/merchantPayment/constants'
+import { useTokenInfoBySymbol } from 'src/tokens/hooks'
+import { fetchTokenBalances } from 'src/tokens/slice'
+import { Currency } from 'src/utils/currencies'
+import Logger from 'src/utils/Logger'
+import { isDekRegisteredSelector } from 'src/web3/selectors'
+
+export default function FeeContainer({ amount }: { amount: BigNumber }) {
+  const LOG_TAG = BASE_TAG + 'FeeContainer'
+
+  const dispatch = useDispatch() as (...args: unknown[]) => void
```

Comment:
i don't think we need this `as` type casting?

Reply:
Remove type casting in 651eed8bb42eee9d8c13004ff2a3f6a262999709!